qase-javascript-commons/package.json

```diff
@@ -26,22 +26,25 @@
   "dependencies": {
     "ajv": "^8.12.0",
     "chalk": "^4.1.2",
+    "child-process-ext": "^3.0.2",
     "env-schema": "^5.2.0",
     "form-data": "^4.0.0",
     "lodash.get": "^4.4.2",
     "lodash.merge": "^4.6.2",
     "lodash.mergewith": "^4.6.2",
+    "mime-types": "^2.1.33",
     "qaseio": "^2.1.0-beta.1",
     "strip-ansi": "^6.0.1",
-    "uuid": "^9.0.0",
-    "child-process-ext": "^3.0.2"
+    "uuid": "^9.0.0"
   },
   "devDependencies": {
     "@jest/globals": "^29.5.0",
     "@types/jest": "^29.5.2",
     "@types/lodash.get": "^4.4.7",
     "@types/lodash.merge": "^4.6.7",
     "@types/lodash.mergewith": "^4.6.7",
+    "@types/mime-types": "^2.1.4",
```

Comment:
Is it not the same as plain `"mime-types": "^2.1.33",` above?

Reply:
yes, I've removed it.

Reply:
hm I got an error without this dependency: 
`Error: src/utils/mimeTypes.ts(2,23): error TS7016: Could not find a declaration file for module 'mime-types'. '/home/runner/work/qase-javascript/qase-javascript/node_modules/mime-types/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/mime-types` if it exists or add a new declaration (.d.ts) file containing `declare module 'mime-types';`
npm ERR! Lifecycle script `build` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: qase-javascript-commons@2.0.0-beta.5 
npm ERR!   at location: /home/runner/work/qase-javascript/qase-javascript/qase-javascript-commons `